BridgeAppSDK/SBAJSONObject.m

```diff
@@ -197,40 +185,40 @@ - (id)jsonObject {
 
 @end
 
-id sba_JSONObjectForObject(id object, NSString * key, NSDictionary <NSString *, NSFormatter *> *formatterMap) {
-    if ([object respondsToSelector:@selector(jsonObjectWithFormatterMap:)]) {
-        return [object jsonObjectWithFormatterMap:formatterMap];
+id SBAJSONObjectForObject(id object) {
+    if (![object isKindOfClass:[NSObject class]]){
+        // If this is not an object that inherits from NSObject class then return NSNull
+        return [NSNull null];
```

Comment:
Alternatively, it could call a Swift function that would see if it was a Swift non-class object (Dictionary, Array, etc.) that it could convert to JSON and assert/return NSNull if it failed.